ArrtSource/Utils/Logging.cpp

```diff
@@ -43,6 +48,40 @@ void ArrtAppWindow::on_ClearLogButton_clicked()
     m_maxLogType = QtDebugMsg;
 }
 
+void ArrtAppWindow::on_ExportLogButton_clicked()
+{
+    bool exported = true;
+    QString fileName = QFileDialog::getSaveFileName(this, tr("Save Log File"),
+                                                    "",
+                                                    tr("Text files (*.txt)"));
+
+    if (fileName.isEmpty() || fileName.isNull())
+    {
+        // the user closed the dialog
+        return;
+    }
+
+    QSaveFile fileOut(fileName);
+    exported &= fileOut.open(QIODeviceBase::WriteOnly);
+    QTextStream out(&fileOut);
+    for (int i = 0; i < LogTab->LogList->count(); i++)
```

Comment:
Does the log mention the ARRT version/ARR client SDK version?
If not, can we put that as first two lines here?

Reply:
fixed!